Frontend/src/main/java/com/harmonia/client/UserClient.java

```diff
@@ -34,13 +40,57 @@ public UserPO addUser(UserPO user) {
         return restTemplate.exchange(USERS_ADD_URL, HttpMethod.POST, request, UserPO.class, urlParameters).getBody();
     }
 
-    public ResponseEntity<Void> removeUser(int userId) {
+    public void removeUser(int userId) {
```

Comment:
Return response entity instead of  void because you need a confirmation that the user was deleted

Reply:
@SampooS